packages/cli/src/Init.ts

```diff
@@ -327,6 +341,99 @@ export class Init implements Command {
     const previewFeatures = args['--preview-feature']
     const output = args['--output']
 
+    if (datasourceProvider === `prisma+postgres`) {
+      /**
+       * Flow:
+       * 1. Check authentication status
+       * 2. If not authed, run login or signup
+       * 3. Find default organization
+       * 4. Create project
+       * 5. Create environment with PPG input
+       * 6. Start polling environment status
+       * 7. Create schema.prisma with PPG data source provider
+       * 8. Create .env.prisma file with PPG URL
+       * 9. Print success message
+       */
+      const PlatformCommands = await import(`./platform/_`)
+
+      const credentials = await credentialsFile.load()
+      if (isError(credentials)) throw credentials
+
+      if (!credentials) {
+        console.log('This will create a project for you on console.prisma.io and requires you to be authenticated.')
+        const authAnswer = await confirm({
+          message: 'Would you like to authenticate?',
+        })
+        if (!authAnswer) {
+          return 'Project creation aborted. You need to authenticate to use Prisma Postgres®'
+        }
+        const authenticationResult = await PlatformCommands.loginOrSignup()
+        console.log(authenticationResult.message)
+      }
+
+      const platformToken = await PlatformCommands.getTokenOrThrow(args)
+      const defaultWorkspace = await PlatformCommands.Workspace.getDefaultWorkspaceOrThrow({ token: platformToken })
+      const regions = await getPrismaPostgresRegionsOrThrow({ token: platformToken })
+
+      const ppgRegionSelection = await select({
+        message: 'Select your region:',
+        default: 'us-east-1',
```

Comment:
(You might want to set this up with fallback for the next available region, in case it's unavailable.)

Reply:
The select component is smart enough. If the default option satisfies the disabled condition, the default value will be changed to next in the list 👍